examples/contour.jl

```diff
@@ -13,3 +36,184 @@ function contour1()
     plot(trace, layout)
 end
+
+function contour4()
+    z = [10     10.625  12.5  15.625  20
+         5.625  6.25    8.125 11.25   15.625
+         2.5    3.125   5.    8.125   12.5
+         0.625  1.25    3.125 6.25    10.625
+         0      0.625   2.5   5.625   10]
+    data = contour(;z=z, colorscale="Jet")
+
+    layout = Layout(;title="Colorscale for Contour Plot")
+    plot(data, layout)
+end
+
+function contour5()
+    z = [10     10.625  12.5  15.625  20
+         5.625  6.25    8.125 11.25   15.625
+         2.5    3.125   5.    8.125   12.5
+         0.625  1.25    3.125 6.25    10.625
+         0      0.625   2.5   5.625   10]
+    data = contour(;z=z,
+                   colorscale="Jet",
+                   autocontour=false,
+                   contours=Dict(:start=>0, :end=>8, :size=>2))
+
+    layout = Layout(;title="Customizing Size and Range of Contours")
+    plot(data, layout)
+end
+
+function contour6()
+    z = [10     10.625  12.5  15.625  20
+         5.625  6.25    8.125 11.25   15.625
+         2.5    3.125   5.    8.125   12.5
+         0.625  1.25    3.125 6.25    10.625
+         0      0.625   2.5   5.625   10]
+    data = contour(;z=z, colorscale="Jet", dx=10, x0=5, dy=10, y0=10)
+
+    layout = Layout(;title="Customizing Spacing Between X and Y Axis Ticks")
+
+    plot(data, layout)
+end
+
+function contour7()
+    z = [[nothing, nothing, nothing, 12, 13, 14, 15, 16],
+        [nothing, 1, nothing, 11, nothing, nothing, nothing, 17],
+        [nothing, 2, 6, 7, nothing, nothing, nothing, 18],
+        [nothing, 3, nothing, 8, nothing, nothing, nothing, 19],
+        [5, 4, 10, 9, nothing, nothing, nothing, 20],
+        [nothing, nothing, nothing, 27, nothing, nothing, nothing, 21],
+        [nothing, nothing, nothing, 26, 25, 24, 23, 22]]
+    trace1 = contour(;z=z, showscale=false, xaxis="x1", yaxis="y1")
+    trace2 = contour(;z=z, connectgaps=true, showscale=false,
+                     xaxis="x2", yaxis="y2")
+    trace3 = heatmap(;z=z, zsmooth="best",showscale=false,
+                     xaxis="x3", yaxis="y3")
+    trace4 = heatmap(;z=z, zsmooth="best", connectgaps=true,
+                     showscale=false, xaxis="x4", yaxis="y4")
+    data = [trace1, trace2, trace3, trace4]
+
+    t = "Connect the Gaps Between Null Values in the Z Matrix"
+    layout = Layout(;title=t,
+                    xaxis=attr(;domain=[0,0.45], anchor="y1"),
+                    yaxis=attr(;domain=[0.55,1], anchor="x1"),
+                    xaxis2=attr(;domain=[0.55,1], anchor="y2"),
+                    yaxis2=attr(;domain=[0.55,1], anchor="x2"),
+                    xaxis3=attr(;domain=[0,0.45], anchor="y3"),
+                    yaxis3=attr(;domain=[0,0.45], anchor="x3"),
+                    xaxis4=attr(;domain=[0.55,1], anchor="y4"),
+                    yaxis4=attr(;domain=[0,0.45], anchor="x4"))
+    plot(data, layout)
+end
+
+function contour8()
```

Comment:
Check out [this section](http://spencerlyon.com/PlotlyJS.jl/manipulating_plots/#subplots) of the documentation for how I avoid setting the axis domain/range information.

`relayout!` simply adjusts fields of the plot's layout. 

Roughly what I do is 

1. create two separate plots, each containing one contour.
2. Use the notation introduced in the doc section I linked to to combine them into one plot with two subplots
3. Adjust the layout on this new plot to have the title we want